packages/core-components/src/components/Table/Filters.tsx

```diff
@@ -24,7 +24,7 @@ import { SelectProps } from '../Select/Select';
 import { coreComponentsTranslationRef } from '../../translation';
 import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
 
-export type TableFiltersClassKey = 'root' | 'value' | 'heder' | 'filters';
+export type TableFiltersClassKey = 'root' | 'value' | 'header' | 'filters';
```

Comment:
Same here, not sure we can do this 🤔

Reply:
I've removed any changes related to `TableFiltersClassKey` from this PR.